internal/resources/package/spec/spec_flatten_test.go

```diff
@@ -50,13 +50,39 @@ func TestFlattenSpecForClusterScope(t *testing.T) {
 						"some1",
 					},
 					RepositoryNameKey: "testRepo",
-					ReleaseNotesKey:   "someReleaseNotes",
-					ReleasedAtKey:     strfmt.DateTime{}.String(),
+					ReleaseNotesKey: []interface{}{
+						map[string]interface{}{
+							metadataNameKey: "cert-manager",
+							versionKey:      "1.1.0",
+							urlKey:          "https://github.com/jetstack/cert-manager/1.1.0",
+						},
+					},
+					ReleasedAtKey: strfmt.DateTime{}.String(),
 					ValuesSchemaKey: []interface{}{
 						map[string]interface{}{
 							TemplateKey: []interface{}{
 								map[string]interface{}{
-									RawKey: "somevalue",
+									RawKey: []interface{}{
+										map[string]interface{}{
+											examplesKey: []interface{}{
+												map[string]interface{}{
+													namespaceKey: "cert-manager",
+												},
+											},
+											propertiesKey: []interface{}{
+												map[string]interface{}{
+													namespaceKey: []interface{}{
+														map[string]interface{}{
+															defaultKey:     "cert-manager",
+															descriptionKey: "The namespace in which to deploy cert-manager.",
+															typeKey:        "string",
+														},
+													},
+												},
+											},
```

Comment:
Check indentation.

Reply:
I've checked it and the indentation seems to be fine. PTAL